src/qml/NightstandWatchfacePage.qml

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (C) 2022 - Timo Könnecke <github.com/eLtMosen>
+ *               2022 - Darrel Griët <dgriet@gmail.com>
+ *               2015 - Florent Revest <revestflo@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.9
+import QtGraphicalEffects 1.12
+import Qt.labs.folderlistmodel 2.1
+import org.asteroid.controls 1.0
+import org.asteroid.utils 1.0
+import Nemo.Configuration 1.0
+import Nemo.Time 1.0
+
+Item {
```

Comment:
Instead of duplicating all this code, we should extract a WatchfaceSelector item that exposes the one or two properties that differ and get instantiated in both WatchfacePage and NighstandWatchfacePage

Reply:
Done!